### ISSUE TITLE
PDE console rendering issue

### ISSUE THREAD

**Issue by zumoshi** [processing/processing#5597]
## Description

`print('A');` displays `!` in console.

## Expected Behavior

`print('A');` would display `A` in console.

## Current Behavior

![image](https://user-images.githubusercontent.com/4184939/43366695-350b681e-9357-11e8-9f74-1fa79c9133b0.png)
selecting/copying text from console copies what was printed, not what is displayed.

## Steps to Reproduce

1. new sketch
2. write `print('A');`
3. run.
(I did not manage to reproduce this issue on a win 7 vm... so please ask for any files/enviromental variables/library or redist software versions etc that could affect this so i can provide more info)

## Your Environment

 

* Processing version: 3.3.7
* Operating System and OS version: Windows 8.1 x64

## Possible Causes / Solutions

no idea what could cause this. since copying the text copies it correctly it seems to be a rendering issue. also, I didn't manage to reproduce this issue on a win 7 vm with the same version. so it could be result of a conflict with other applications on my system.

I can provide any other info that you think might help with diagnosing the issue.

**Comment by gohai** [comment]
Never seen anything like this. What happens if you change the "Editor & Console Font" setting in Preferences?

**Comment by zumoshi** [comment]
I thought it could be a font thing too, but since editor and console use the same font, it doesn't make sense for the editor to be fixed and console be broken, either both should show gibberish or none.

as for your question, I'm probably missing something but the option is greyed out.
![image](https://user-images.githubusercontent.com/4184939/43369327-50006384-9381-11e8-9f71-af77bc28d450.png)

**Comment by zumoshi** [comment]
nvm, I managed to change it. it actually got fixed when I changed it to `simplified Arabic fixed`:
![image](https://user-images.githubusercontent.com/4184939/43369339-88dd5324-9381-11e8-8085-928588a1736e.png)
but I can reproduce the error by setting the font back to `source code pro` or any of its variants

**Comment by gohai** [comment]
@zumoshi I see, so it must be something else. Thanks for checking and glad you're able to use it now at least.

**Comment by zumoshi** [comment]
yeah, thanks for the workaround. I was copy-pasting text from console to notepad++ to read debug messages each time before...

feel free to ask any other questions. i can even give you remove access to my system if you wanted to check it out yourself.
